src/runtime/security/kinit_context.cpp

```diff
@@ -45,6 +45,8 @@ DSN_DECLARE_bool(enable_auth);
 DSN_DEFINE_string("security", krb5_keytab, "", "absolute path of keytab file");
 DSN_DEFINE_string("security", krb5_config, "", "absolute path of krb5_config file");
 DSN_DEFINE_string("security", krb5_principal, "", "kerberos principal");
+DSN_DEFINE_string("security", service_fqdn, "", "service fqdn");
```

Comment:
How about adding some comments about the usage of `fqdn`?

Reply:
I add it to the desc param of `DSN_DEFINE_string`